### DISCUSSION TITLE
What am I doing wrong?  No mesh or videos are being created.

### DISCUSSION THREAD

**Discussion by keenerb** [thygate/stable-diffusion-webui-depthmap-script#347] [General]
![image](https://github.com/thygate/stable-diffusion-webui-depthmap-script/assets/1183671/c7646f27-0825-47ca-8f63-662e7d32fa23)
![image](https://github.com/thygate/stable-diffusion-webui-depthmap-script/assets/1183671/c09166da-723a-4578-b69e-3ddefa426eb1)

Nothing seems to be created for .obj or animations.

**Comment by keenerb** [comment]
Works through the DEPTH tab.  The SCRIPT option to generate a mesh doesn't seem to do anything.

**Comment by graemeniedermayer** [comment]
I created an issue for this here https://github.com/thygate/stable-diffusion-webui-depthmap-script/issues/348#issue-1918561850

Thanks for bring this up! It might be UI issue (i'm not sure that option should be available in the script mode).
